app/actions/usersActivity.js

```diff
@@ -0,0 +1,24 @@
+import { USERS_ACTIVITY } from './actionsTypes';
+
+export function addUserActivity(username, activity, rid, tmid) {
+	return {
+		type: USERS_ACTIVITY.ADD,
+		roomId: tmid || rid,
+		username,
+		activity
+	};
+}
+
+export function clearUserActivity(username, roomId) {
```

Comment:
What about we do the logic as `addUserActivity` and send `rid, tmid` as params, and `roomId: tmid || rid`?